java/test/jmri/util/JUnitUtil.java

```diff
@@ -428,7 +428,10 @@ public String getSystemPrefix() {
             }
 
         };
-        InstanceManager.setDefault(jmri.CommandStation.class, cs);
+        // we should use setDefault here, but setCommandStation will
```

Comment:
When unit testing, do we care if a consist manager is in place if we aren't testing one?

Reply:
No, but when performing integration tests we do.